os/board/common/Kconfig

```diff
@@ -0,0 +1,51 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config SFLASH
+	bool "SFLASH"
+	default y
+	---help---
+		Use FLASH
+
+config FLASH_PART

```

Comment:
1. I can't imagine the `partition` from `FLASH_PART`. 
2. This is a main config of this functionality.
Because of above, let's use a full word, `PARTITION`.